sentence_transformers/SentenceTransformer.py

```diff
@@ -30,6 +30,27 @@
 
 logger = logging.getLogger(__name__)
 
+
+class HubParameters:
```

Comment:
Maybe this should be renamed to `HubTrainingArguments`? My concern if not is that if you use `save_to_hub` with a `SentenceTransformers` model that was initialized with params for a different repo/org, then the behaviour becomes confusing. And if we do that maybe `HubTrainingArguments` should just be passed in `fit`. WDYT?

Reply:
Sure, that sounds good to me! I'll remove it from the SentenceTransformers constructor.